docs/spec/auth/token.md

```diff
@@ -55,19 +55,20 @@ The new registry workflow is more like this:
    HTTP response with information on how to authenticate.
 3. The registry client makes a request to the authorization service for a
    signed JSON Web Token.
-4. The authorization service returns a token.
+4. The authorization service returns a token with a subset of the requested
```

Comment:
I don't think this should be restricted to a "subset". A superset of the permissions may be valid, as well. Perhaps, this should read:

    The authorization service returns an opaque token representing the available grants to the client.